packages/app-bridge-theme/src/types/Guideline.ts

```diff
@@ -56,53 +63,59 @@ export interface GuidelineDocumentLink {
     parentId(): Nullable<number>;
     type: 'document-link';
 }
+export interface DocumentLinkNavigationItem extends DocumentLinkBase {}
 
-export interface GuidelinePageCategory {
+interface PageCategoryBase {
     id(): number;
     title(language?: string): string;
     slug(language?: string): string;
-    children(): (GuidelineDocumentPage | GuidelineDocumentPageLink)[];
+    children(): (DocumentPageNavigationItem | DocumentPageHeadingNavigationItem)[];
     type: 'page-category';
 }
+export interface PageCategoryNavigationItem extends PageCategoryBase {}
 
-export interface GuidelineDocumentPage {
+interface DocumentPageBase {
     id(): number;
     title(language?: string): string;
     slug(language?: string): string;
     url(language?: string): string;
-    headings(): GuidelineDocumentPageHeading[];
     type: 'document-page';
 }
+export interface DocumentPageNavigationItem extends DocumentPageBase {
+    headings(): DocumentPageHeadingNavigationItem[];
+}
+export interface DocumentPage extends DocumentPageBase {}
 
-export interface GuidelineDocumentPageLink {
+interface DocumentPageLinkBase {
     id(): number;
     title(language?: string): string;
     url(): string;
     type: 'document-page-link';
 }
+export interface DocumentPageLinkNavigationItem extends DocumentPageLinkBase {}
 
-export interface GuidelineDocumentPageHeading {
+interface DocumentPageHeadingBase {
     id(): number;
     title(language?: string): string;
     slug(language?: string): string;
     type: 'document-page-heading';
 }
+export interface DocumentPageHeadingNavigationItem extends DocumentPageHeadingBase {}
```

Comment:
IMO we don’t need the `DocumentPageHeadingBase` for now. Once we need a `DocumentPageHeading` we can easily add it. Wdyt?